_config.yml

```diff
@@ -43,6 +43,7 @@ plugins:
   - jekyll-titles-from-headings
   - jekyll-relative-links
   - jekyll-avatar
+  - jekyll-github-metadata

```

Comment:
This was already added on line 39. TBH I don't really understand the difference between adding in the gem and adding in the `_config.yml`. Are both required?

Reply:
It works locally but not on CI/CD.

Reply:
As for difference it depends on version of jeckyll but yes. this PR did not made any fix. Sorry